.gitlab-ci.yml

```diff
@@ -0,0 +1,4 @@
+# DESCRIPTION: GitLab CI/CD for libRetro (NOT FOR GitLab-proper)
+
+include:
+  - 'backends/platform/libretro/.gitlab-ci.yml'
```

Comment:
As discussed on Discord, since the main ScummVM is not using gitlab, this file is OK to stay